libs/core/langchain_core/runnables/fallbacks.py

```diff
@@ -89,6 +91,11 @@ def when_all_is_lost(inputs):
     
     Any exception that is not a subclass of these exceptions will be raised immediately.
     """
+    exception_key: Optional[str] = None
```

Comment:
Could you update the doc-string to explain how to interpret this key?

It's unclear from the documentation how this key interacts with exceptions_to_handle.

Is it always passed to the runnable or only for exceptions that are defined in the list of exceptions to handle or only for  unhandled exceptions?